resources/psalm.json

```diff
@@ -5,29 +5,58 @@
             "summary": "Finds errors in PHP applications",
             "website": "https://psalm.dev/",
             "command": {
-                "file-download": {
-                    "url": "https://github.com/vimeo/psalm/releases/download/4.6.4/psalm.phar.asc",
-                    "file": "%target-dir%/psalm.phar.asc"
-                },
-                "phar-download": {
-                    "phar": "https://github.com/vimeo/psalm/releases/download/4.6.4/psalm.phar",
-                    "bin": "%target-dir%/psalm"
+                "composer-bin-plugin": {
+                    "package": "vimeo/psalm",
+                    "namespace": "psalm",
+                    "links": {
+                        "%target-dir%/psalm": "psalm",
+                        "%target-dir%/psalm-language-server": "psalm-language-server",
+                        "%target-dir%/psalm-plugin": "psalm-plugin",
+                        "%target-dir%/psalm-refactor": "psalm-refactor",
+                        "%target-dir%/psalter": "psalter"
+                    }
                 }
             },
             "test": "psalm -h",
             "tags": ["featured", "psalm"]
         },
         {
-            "name": "doctrine-psalm-plugin",
+            "name": "psalm-plugin-doctrine",
             "summary": "Stubs to let Psalm understand Doctrine better",
             "website": "https://github.com/weirdan/doctrine-psalm-plugin",
             "command": {
                 "composer-bin-plugin": {
                     "package": "weirdan/doctrine-psalm-plugin",
-                    "namespace": "doctrine-psalm-plugin"
+                    "namespace": "psalm"
+                }
+            },
+            "test": "rm -f composer.json; psalm-plugin show | grep weirdan/doctrine-psalm-plugin",
```

Comment:
`psalm-plugin` works if
1. there's a `composer.json` and `vendor/`
2. both are missing
In our case, we have just `composer.json` so it's saying there should be an autoloader here too, even there is Psalm's one already.

Not sure what the better choice here would be, maybe changing to the Psalm namespace root and running it there?

Reply:
> Not sure what the better choice here would be, maybe changing to the Psalm namespace root and running it there?

Either this, or change to a tmp (or root) directory.

Reply:
@jakzal done.